apps/workshop/projects/DavosProtocol/modules/DUSDStake.ts

```diff
@@ -0,0 +1,64 @@
+import type { ModuleDefinitionInterface } from '@defiyield/sandbox';
+
+import stDUSDAbi from '../abis/stDUSD.abi.json';
+
+const stDUSDAddress = '0xe69a1876bdacfa7a7a4f6d531be2fde843d2165c';
+
+export const DUSDStake: ModuleDefinitionInterface = {
+  name: 'DUSDStake',
+  chain: 'polygon',
+  type: 'staking',
+
+  async preloadTokens() {
+    return [stDUSDAddress];
+  },
+
+  async fetchPools({ tokens, ethcallProvider, ethcall, BigNumber, axios }) {
+    const [token] = tokens;
+    const contract = new ethcall.Contract(stDUSDAddress, stDUSDAbi);
+
+    const [totalSupply] = await ethcallProvider.all<typeof BigNumber>([contract.totalStaking()]);
+    const tvl = (Number(totalSupply.toString()) / 10 ** token?.decimals) * (token?.price || 0);
+
+    let apr = 0;
+    try {
+      const response = await axios.get('https://stake.ly/api/stats/stDUSD/apr');

```

Comment:
Does the URL work fine? 
<img width="1064" alt="image" src="https://github.com/de-dot-fi/defiyield-self-integrations/assets/100587071/518f4c7c-9ce1-49b2-9057-85e7678681aa">